src/bindings.zig

```diff
@@ -0,0 +1,52 @@
+const std = @import("std");
+const gpu = @import("gpu");
+const Core = @import("Core.zig");
+const libmach = @import("platform/libmach.zig");
+const native = @import("platform/native.zig");
+
+// Current Limitations:
+// 1. Currently, ecs seems to be using some weird compile-time type trickery, so I'm not exactly sure how
+// `engine` should be integrated into the C API
+// 2. Core might need to expose more state so more API functions can be exposed (for example, the WebGPU API)
```

Comment:
Once https://github.com/hexops/mach/pull/403 lands we should end up being ABI-compatible with the `webgpu.h` API, and Zig's stage2 compiler could even emit `webgpu.h` for us based on this too I think. Should be possible to start writing Mach core / WebGPU apps using this.